pkg/network/http/testutil/testutil.go

```diff
@@ -105,7 +105,7 @@ func StatusFromPath(path string) (status int) {
 }
 
 func CurDir() (string, error) {
-	_, file, _, ok := runtime.Caller(0)
+	_, file, _, ok := runtime.Caller(1)
```

Comment:
why did you change it?

Reply:
please make sure the change did not cause any test to fail

Reply:
CI is passing.